Vagrantfile

```diff
@@ -145,6 +145,11 @@ Vagrant.configure(2) do |config|
       fi
     fi
 
+    # Make sure Docker is running before running any scripts that rely on it.
```

Comment:
@f0rmiga Have you seen the interesting while loop a few lines above your changes ?

While this is conditional on the registry mirror setting, it does mention Virtualbox as a source of problems and works quite hard on restarting docker.

You might be forced to loop as well instead of one-shooting it.

Another possible fix would be to move the entire loop after the conditional, replacing your change, and further dropping the `if ~ sysctl restart ...` from the conditional. Just have it stop docker, and the unconditional loop after would restart it anyway, seeing it as not active.

Reply:
I was a bit hesitant to change the code above. I'll do it after the OP gives the feedback.

Reply:
Makes sense.